### PR TITLE
Remove false positives with RawURL

### PR DESCRIPTION
**Pull request by isuruf**
Fixes https://github.com/regro/cf-scripts/issues/322
Fixes https://github.com/regro/cf-scripts/issues/476

### CODE REVIEW

File: conda_forge_tick/update_upstream_versions.py [hunk 1]
```diff
@@ -152,6 +153,18 @@ def get_version(self, url):
         return str(ver).replace("-", "_")
 
 
+def get_sha256(url):
```
Comment (CJ-Wright) [inline conda_forge_tick/update_upstream_versions.py]: Would it be worthwhile to use https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators.xsh#L272 here?

Comment (isuruf) [reply conda_forge_tick/update_upstream_versions.py]: I'd like to keep this file pure python if you don't mind

Comment (CJ-Wright) [reply conda_forge_tick/update_upstream_versions.py]: You can import from xonsh files within python, `from rever.tools import hash_url` should work within python.

Comment (isuruf) [reply conda_forge_tick/update_upstream_versions.py]: Sure, but I usually import this file in ipython to debug it and it works with only a minor change. I'd like to keep my development workflow the same if it's not too much trouble.

Comment (CJ-Wright) [reply conda_forge_tick/update_upstream_versions.py]: Ok

Comment (scopatz) [reply conda_forge_tick/update_upstream_versions.py]: I don't really understand what the workflow difference is.  You could still import this file in IPython and debug it...

**Comment by isuruf** [comment]
Can we merge this please?

**Comment by scopatz** [comment]
Hey @isuruf - we really like the idea here!

@CJ-Wright and I think this is duplicating code in a 3rd party library (rever), that we don't feel there is a need to duplicate.  So it is more about the implementation than the idea. 

It would be great if this could use the API from rever or push this down into rever!

File: conda_forge_tick/update_upstream_versions.py (hunk 1 above)
Comment (isuruf) [reply conda_forge_tick/update_upstream_versions.py]: I don't have a xonsh installation and please don't force me to do so.

**Comment by isuruf** [comment]
ping. the bot keeps making incorrect PRs at https://github.com/conda-forge/qd-feedstock

**Comment by jakirkham** [comment]
Thanks for doing this @isuruf! 😄

**Pull request merged**